app/views/posts/_form.html.erb

```diff
@@ -21,7 +21,7 @@
   </div>
   <div class="field">
     <%= f.label :author %><br>
-    <%= f.number_field :author %>
+    <%= f.label User.get_username_by_id(@post.author) %>
```

Comment:
Is this temporary? Why do they need to specify the author in the form? Shouldn't it use the logged in user?